packages/lib-user/package.json

```diff
@@ -66,31 +65,27 @@
     "@testing-library/dom": "~9.3.0",
     "@testing-library/react": "~14.2.0",
     "@testing-library/user-event": "~14.5.0",
+    "@vitejs/plugin-react": "~4.4.1",
     "@zooniverse/grommet-theme": "~3.2.0",
     "@zooniverse/react-components": "~1.13.0",
     "babel-loader": "~10.0.0",
     "babel-plugin-module-resolver": "~5.0.0",
     "babel-plugin-styled-components": "~2.1.4",
     "babel-plugin-transform-imports": "~2.0.0",
-    "chai": "~4.5.0",
-    "chai-dom": "~1.12.0",
     "css-loader": "~7.1.1",
-    "dirty-chai": "~2.0.1",
     "eslint-plugin-jsx-a11y": "~6.10.1",
     "eslint-plugin-react": "~7.37.2",
     "html-webpack-plugin": "~5.6.0",
     "jsdom": "~26.1.0",
-    "mocha": "~11.4.0",
     "msw": "~2.7.0",
     "msw-storybook-addon": "~2.0.3",
     "nock": "~13.5.4",
     "process": "~0.11.10",
     "prop-types": "^15.8.1",
-    "sinon": "~20.0.0",
```

Comment:
Oops I think `sinon` is not supposed to be removed as a devDep. I'll fix before this PR is merged!